examples/huge-output-command/.gitignore

```diff
@@ -0,0 +1 @@
+*.gvmi
```

Comment:
if anything, this should be added to the global `.gitignore` ... but I wouldn't add it at all ...

Reply:
I don't understand what you want me to do then. Do you want me to upload the image? It has almost 5mb

Reply:
no, what I meant is that if we need this ignore to be present, it should be added to the `.gitignore` file in the root of the repository... but on the other hand, you can also add it to your local ignores so that it doesn't show on your listings ... 

by no means I wanted you to upload a GMVI file into the repo :D